services/types.go

```diff
@@ -1,7 +1,16 @@
 package services
 
+type Role string
+
+const (
+	ClientRole        Role = "client"
+	MirroringPeerRole Role = "mirroring-peer"
```

Comment:
The rule is not related to mirroring. 
As for the agreed architecture, there are 2 rules: `ocs-client` and `ocs-peer`,

Please align everything with the given terminology